README.md

```diff
@@ -64,17 +64,9 @@ eg.
 > export MAJOR_TIMEZONES=["UTC", "Asia/Kolkata"]
```

Comment:
This should be MAJOR_TIMEZONES=["UTC","Asia/Kolkata"], Please make this small change, rest LGTM. Thanks

Reply:
done